book/website/_toc.yml

```diff
@@ -380,6 +380,13 @@ parts:
         file: communication/social-media/social-media-twitter-tips
       - title: Managing multiple twitter accounts
         file: communication/social-media/social-media-twitter-multiple
+    - title: Research Objects in Action
+      file: communication/research-objects
+      sections:
+        - title: Concepts
+          file: communication/research-objects/ro-concepts
+        - title: Implementation
```

Comment:
It looks like the page on 'platforms' still needs to be added in the TOC?

Reply:
So platforms is the same as implementation? Or do you want to add more information on the implementation and is platforms only a subheader? It is a bit confusing to me right now!

Reply:
@EstherPlomp thanks for your comment, we've renamed the subheader to **Implementation and associated platforms**. see commit 1e5016fa976245235f399c7e7e11cb210dc5dacf